doc/examples/segmentation/plot_rolling_ball.py

```diff
@@ -0,0 +1,188 @@
+"""
+================================================================
+Use the Rolling Ball Method for Background Intensity Estimation
+================================================================
+
+The rolling ball algorithm estimates background intensity of a grayscale
+image in case of uneven exposure. It is frequently used in biomedical
+image processing and was first proposed by Stanley R. Sternberg (1983) in
+the paper Biomedical Image Processing [1]_.
+
+The idea of the algorithm is quite intuitive. We think of the image as a
+surface that has unit-sized blocks stacked on top of each other for each
+pixel. The number of blocks is determined by the intensity of a pixel. To get
+the intensity of the background at a desired position, we imagine submerging a
+ball into the blocks at the desired pixel position. Once it is completely
+covered by the blocks, the height of the ball determines the intensity of the
+background at that position. We can then *roll* this ball around below the
+surface to get the background values for the entire image. As Sternberg
+puts it:
+
+    "We can visualize a solid sphere that moves freely within the solid
+    volume of the gel image umbra but is constrained by the upper surface of
```

Comment:
I find your explanations clearer than Sternberg's :-). In particular, the "gel image umbra" is quite mysterious to me.

Reply:
I removed the quote. If you find it less clear, then it doesn't add to the example and we should try to keep things short and snappy.